data_model/1.3/clusters/Binding-Cluster.xml

```diff
@@ -94,12 +96,14 @@ Davis, CA 95616, USA
     </struct>
   </dataTypes>
   <attributes>
-    <attribute id="0x0000" name="Binding" type="list" default="[]">
+    <attribute id="0x0000" name="Binding" type="list" default="empty">
```

Comment:
Is the word "empty" here really expected?

Reply:
https://github.com/project-chip/alchemy/issues/23